Network/HTTP/Req.hs

```diff
@@ -380,7 +382,8 @@ import qualified Web.Authenticate.OAuth       as OAuth
 -- >   liftIO $ print (responseBody response :: Value)
 
 req
-  :: ( MonadHttp    m
+  :: forall m method body response scheme.
```

Comment:
I am not sure if this is the "correct" (i.e. most convenient) way to order the arguments.

Reply:
I just realised that this is also a "breaking change".